### PR TITLE
Lesson 2 - Record ready

### PR DESCRIPTION
**Pull request by tomicapretto**
The great majority of the lesson is ready to be recorded I think. I would like your reviews @canyon289 and @AlexAndorra. 

I think we're missing something related to predictions, like what is done in the introductory course. I think it would allow us to cover two things at the same time: how to do predictions and how to obtain business-oriented insights. What do you think?

### CODE REVIEW

**Comment by canyon289** [comment]
@tomicapretto I only did a very fast review without looking at things in slides. I will give much more in depth review in next 48 hours.

Overall the material is fantastic, pacing and narration look great, and your notes are all asking great questions. It looke slike some slides might be text heavy which might be fine, especially if we include animations or something for some visual flair.

**Comment by canyon289** [comment]
Note: I made most of these fixes on this PR https://github.com/tomicapretto/etc/pull/9

## Picture should be larger
image

## Make the bullet point fragments
Theres so much text on screen at once. Either make the bullet point fragements, or remove entirely and say the bullet points, and we can have Fadi highlight the numbers as you're speaking for visual effect.

I slightly prefer the second option. Will leave it up to you
image

## Reword the section summary here
Smoothing out the summary so its more concise, the points are more precise, and it "flows" from what were doing in this lesson, to best practice overall
image

```
* We'll build our model one step at a time in the following steps
    * No predictors
    * A single predictor
    * Two predictors (length + species)
    * How to keep growing it
* This is the Bayesian Workflow
  * Modeling workflow are smoother when deliberately planned   as opposed to random guessing
  * Following the workflow will save you time
  * Complexity is only added when *needed*
```

## Merge the bullet block into the spoken notes
Its both messing up the formatting of the slide, and also too much text for a section intro slide
image

**Comment by tomicapretto** [comment]
@canyon289 thanks so much for all the feedback and highly valuable proposals and modifications. They really help to make this much better. I added a couple of last modifications. I think it's 98/99% done. Just need some more narrative and explanations in the last sections and I'm ready to start recording. 

Could you have a final review? Thanks!

**Comment by canyon289** [comment]
of course

**Comment by canyon289** [comment]
Will review in next 24 hours

**Comment by canyon289** [comment]
This is nearly there. Primarily

* Intro and lesson recap need to built out
* Last section needs talking notes and a more clear takeaway that models can get out of control and though pymc is easier things can be even easier

File: lessons/02_RegressionRefresher/Regression_Refresher_Exercises.ipynb [hunk 1]
```diff
@@ -0,0 +1,183 @@
+{
```
Comment (canyon289) [inline lessons/02_RegressionRefresher/Regression_Refresher_Exercises.ipynb:1]: <p>This is a great start. Add the solutions in here as well as thats a common request from students. No need to be overly detailed.</p><p> </p><p>Once the solutions are in I can split this into two notebooks and then well be done!</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tomicapretto/etc/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

**Comment by tomicapretto** [comment]
<p>Brilliant idea! Let's do it!</p>

---

 <a href='https://app.reviewnb.com/tomicapretto/etc/pull/7/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by tomicapretto** [comment]
<p>I was so dumb! I forgot to pass the <code>observed</code> argument to the likelihood. After passing it, it looks so much better.</p><p> </p><p>Do you still think we could split the exercise in two, so we can use it in lesson 2 and 3? </p>

---

 <a href='https://app.reviewnb.com/tomicapretto/etc/pull/7/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by canyon289** [comment]
<p>Haha classic. I think splitting it in two is a still a good idea. Better to start easy in lesson 2, and then in lesson 3 make an exercise thats</p><p> </p><ol><li>Do the concrete regression in PyMC and add in all the covariates, and also mean center these 3. Omg what a headache</li><li>Now do it bambi, Wow sooooo much easier</li></ol>

---

 <a href='https://app.reviewnb.com/tomicapretto/etc/pull/7/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by canyon289** [comment]
This has been merged and added to main repo. Closing

**Pull request closed without merging**